00_organization/rse_basics_slides.md

```diff
@@ -61,7 +61,7 @@ slideOptions:
 
 - *"Movement"* started in the UK, first UK RSE conference in 2016
 - First conferences in Germany and the Netherlands in 2019
-- [de-RSE position paper](https://f1000research.com/articles/9-295/v2) in 2020
+- [first de-RSE position paper](https://f1000research.com/articles/9-295/v2) in 2020
```

Comment:
You can also cite the teachingRSE (Foundational competencies) paper if you want: https://de-rse.org/blog/2024/10/08/identifying-the-foundational-competencies-of-an-RSE-en.html

There are also multiple working groups now: https://de-rse.org/en/working_groups.html

```suggestion
- [first de-RSE position paper](https://f1000research.com/articles/9-295/v2) in 2020, [several working groups](https://de-rse.org/en/working_groups.html)
   - [learning and teaching RSE](https://de-rse.org/learn-and-teach/), [foundational competencies of an RSE](https://de-rse.org/blog/2024/10/08/identifying-the-foundational-competencies-of-an-RSE-en.html)
```